trl/scripts/sft.py

```diff
@@ -104,7 +104,8 @@ def main(script_args, training_args, model_args):
 
     # Set default chat template if needed
     if tokenizer.chat_template is None:
-        model, tokenizer = setup_chat_format(model, tokenizer, format="chatml")
+        # TODO: source should be passed as an argument
+        model, tokenizer = clone_chat_template(model, tokenizer, "Qwen/Qwen3-0.6B")
```

Comment:
We have found it useful internally to expose a `chat_template` arg in `SFTConfig` which allows one to define a custom template or copy-paste one from an existing model. Perhaps we could expose this along with a `chat_template_clone` arg (or something similar), now that I better understand what your intent was in this PR?

Reply:
sounds good!

Reply:
#3599